### PR TITLE
Place the MQTT Loop in a try/except block

### PR DESCRIPTION
**Pull request by makermelissa**
This is to solve an issue that both @jedgarpark and I ran into when running this overnight:
```
Traceback (most recent call last):
  File "code.py", line 65, in <module>
  File "adafruit_io/adafruit_io.py", line 215, in loop
  File "adafruit_minimqtt/adafruit_minimqtt.py", line 782, in loop
  File "adafruit_minimqtt/adafruit_minimqtt.py", line 552, in ping
MMQTTException: PINGRESP not returned from broker.
```

### CODE REVIEW

**Review by ladyada — APPROVED** [review]

File: adafruit_funhouse/network.py [hunk 1]
```diff
@@ -107,6 +105,14 @@ def _get_mqtt_client(self):
             return self._mqtt_client
         raise RuntimeError("Please initialize MQTT before using")
 
+    def mqtt_loop(self):
+        """Run the MQTT Loop"""
+        try:
+            if self._mqtt_client is not None:
+                self._mqtt_client.loop()
+        except MQTT.MMQTTException:
+            pass
```
Comment (dhalbert) [inline adafruit_funhouse/network.py]: `except`-`pass` makes any failure invisible, which is hard to debug. Could you print something, or at least return a boolean success/failure?

**Review by dhalbert — CHANGES_REQUESTED** [review]

File: adafruit_funhouse/network.py (hunk 1 above)
Comment (makermelissa) [reply adafruit_funhouse/network.py]: Sure

**Comment by brentru** [comment]
@makermelissa  I didn't get to initially review but just looked thru this...
`return self.init_mqtt(IO_MQTT_BROKER, 1883, aio_username, aio_key, True)`

May want to use secure port `8883`. Setting up the client for the S2/SSL will be different and may need to be modified. I can take out another issue.

File: adafruit_funhouse/network.py (hunk 1 above)
Comment (dhalbert) [reply adafruit_funhouse/network.py]: Thanks - I hope it will not flood the output if it retries frequently.

**Review by dhalbert — APPROVED** [review]
:+1: though do see @brentru's comment.

**Comment by makermelissa** [comment]
@brentru let's do a new issue for that.

**Pull request merged**